.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: c
+dist: trusty
```

Comment:
@agramfort I don't think so. Maybe some minutes on each container. But it's not enough. I'll be looking at the tests in subsequent PRs.